packages/strapi-plugin-content-manager/admin/src/containers/EditViewLayoutManager/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { LoadingIndicatorPage } from 'strapi-helper-plugin';
+import EditView from '../EditView';
+import { resetProps, setLayout } from './actions';
+import selectLayout from './selectors';
+
+const EditViewLayoutManager = ({ layout, ...rest }) => {
+  const currentLayout = useSelector(selectLayout);
+  const dispatch = useDispatch();
+
+  console.log({ layout });
```

Comment:
console.log

Reply:
The log will be removed in the other PRs.